src/commonMain/rust/bridge/continue_watching_preview.rs

```diff
@@ -0,0 +1,43 @@
+use stremio_core::deep_links::LibraryItemDeepLinks;
```

Comment:
Probably it's better to name this file as `ContinueWatchingItem` to match what it's working with.